src/test/java/org/opentripplanner/middleware/ApiUserFlowTest.java

```diff
@@ -35,19 +35,29 @@
 import static org.opentripplanner.middleware.auth.Auth0Connection.isAuthDisabled;
 import static org.opentripplanner.middleware.auth.Auth0Users.createAuth0UserForEmail;
 import static org.opentripplanner.middleware.controllers.api.ApiUserController.DEFAULT_USAGE_PLAN_ID;
-import static org.opentripplanner.middleware.controllers.api.OtpRequestProcessor.OTP_PLAN_ENDPOINT;
 import static org.opentripplanner.middleware.controllers.api.OtpRequestProcessor.OTP_PROXY_ENDPOINT;
-import static org.opentripplanner.middleware.persistence.PersistenceUtil.createMonitoredTrip;
+import static org.opentripplanner.middleware.otp.OtpDispatcher.OTP_PLAN_ENDPOINT;
 
 /**
  * Tests to simulate API user flow. The following config parameters must be set in configurations/default/env.yml for
- * these end-to-end tests to run: - AUTH0_DOMAIN set to a valid Auth0 domain - AUTH0_API_CLIENT set to a valid Auth0
- * application client id - AUTH0_API_SECRET set to a valid Auth0 application client secret - DEFAULT_USAGE_PLAN_ID set
- * to a valid usage plan id (AWS requires this to create an api key) - OTP_API_ROOT set to http://localhost:8080/otp so
- * the mock OTP server is used - OTP_PLAN_ENDPOINT set to /plan - An AWS_PROFILE is required, or AWS access has been
- * configured for your operating environment e.g. C:\Users\<username>\.aws\credentials in Windows or Mac OS equivalent.
+ * these end-to-end tests to run:
  *
- * The following environment variable must be set for these tests to run: - RUN_E2E=true
+ * AUTH0_DOMAIN set to a valid Auth0 domain.
+ *
+ * AUTH0_API_CLIENT set to a valid Auth0 application client id.
+ *
+ * AUTH0_API_SECRET set to a valid Auth0 application client secret.
+ *
+ * DEFAULT_USAGE_PLAN_ID set to a valid usage plan id (AWS requires this to create an api key).
+ *
+ * OTP_API_ROOT set to a live OTP instance (e.g. http://otp-server.example.com/otp).
```

Comment:
This is no longer accurate... is this the result of a bad merge?

Reply:
@landonreed yes I think so. I had updated so the tests would be against a live OTP server and removed the `TestUtils.mockOtpServer()` call from the `setUp()` method (which is now back). This was in response to "TODO: Run a live OTP instance?".

Which is the idea approach, mock or live OTP server?